docs/conf/portland/2018/tickets.rst

```diff
@@ -4,7 +4,7 @@
 Tickets
 =======
 
-**Tickets on sale now!** Follow the links below for each ticket type. 
+**Tickets on sale now!**
```

Comment:
I'd put a button or call out of some kind here, so people who want can just click through.

Reply:
I believe we should already have some defined in the HTML/CSS somewhere. Can probably just steal it from the index, unless the CSS styles are super targeted.

Reply:
![cta](https://user-images.githubusercontent.com/95874/34688685-400a5e84-f4b3-11e7-9fe4-a87917f27844.png)

Looks kinda meh if you just throw it straight in

Reply:
I guess I could live with it floated right and vertical spacing sorted. 

Thoughts?